pkg/ebpf/driver_interface.go

```diff
@@ -0,0 +1,373 @@
+// +build windows
+
+package ebpf
+
+/*
+//! These includes are needed to use constants defined in the ddfilterapi
+#include <WinDef.h>
+#include <WinIoCtl.h>
+
+//! Defines the objects used to communicate with the driver as well as its control codes
+#include "c/ddfilterapi.h"
+*/
+import "C"
+import (
+	"encoding/binary"
+	"fmt"
+	"net"
+	"sync"
+	"sync/atomic"
+	"unsafe"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+
+	"golang.org/x/sys/windows"
+)
+
+// HandleType expresses what of DriverHandle one is - implicitly implies if filters have been placed on the driver for the handle
+type HandleType string
+
+const (
+	driverFile = `\\.\ddfilter`
```

Comment:
Is this actually a file path?

Reply:
That's a good point - I'm going to ping Derek to see if this is an accurate name. I suspect that something like "devicePath" might be more accurate windows terminology.

Reply:
Derek suggested `deviceName` so going to update it to that and add a comment